awx/ui/client/features/output/_index.less

```diff
@@ -1,7 +1,7 @@
 @import 'host-event/_index';
 .at-Stdout {
     &-menuTop {
-        color: @at-gray-848992;
+        color: @at-gray-646972;
```

Comment:
This is too light for labels throughout the UI.

Reply:
![image](https://user-images.githubusercontent.com/214912/49091784-86df5200-f22e-11e8-95f9-850a3aa33463.png)

vs

![image](https://user-images.githubusercontent.com/214912/49091665-5697b380-f22e-11e8-968d-69de27aacc9f.png)